jsconfig.json

```diff
@@ -2,17 +2,17 @@
 	"compilerOptions": {
 		"checkJs": true,
 		"allowSyntheticDefaultImports": true,
-		"module": "commonjs",
 		"target": "es2017",
+		"module": "es6",
 		"baseUrl": ".",
 		"paths": {
 			"*": [ "test/*", "server/*", "client/*" ]
 		},
 		"jsx": "preserve"
 	},
-	"exclude": [
-		"node_modules",
-		"public",
-		"build"
+	"include": [
```

Comment:
https://www.typescriptlang.org/docs/handbook/module-resolution.html#path-mapping

it doesn't really relate. `include` tells typescript what to consider a valid source file. `paths` tells is where to look for required files that are not relative requires. A star mapping like this is sort of like NODE_PATH in node land.